Dockerfile.dist

```diff
@@ -0,0 +1,4 @@
+FROM scratch
```

Comment:
Yeah, the primary Dockerfile is setup for build, and the new one is for distribution. The differences are dramatic.

 From what I have seen on Google, and our own stuff with sherdock this approach seems to be the way to go.